tests/spec/core/markdown-spec.js

```diff
@@ -8,14 +8,18 @@ describe("Core - Markdown", function() {
     var ops = {
       config: makeBasicConfig(),
       body: makeDefaultBody() +
-       "\nFoo\n===\n",
+       "\n\nFoo\n===\n",
```

Comment:
Ok, this is kinda f'ed up...  
```JS
// Test 1 - what we currently 
marked('<section id="abstract"><p>test abstract</p></section><section id="sotd">\n\nFoo\n===')
"<p><section id="abstract"><p>test abstract</p></section><section id="sotd"></p>
<h1 id="foo">Foo</h1>
"
// Test 2 - original
marked('<section id="abstract"><p>test abstract</p></section><section id="sotd">\nFoo\n===')
"<p><section id="abstract"><p>test abstract</p></section><section id="sotd"></p>
<h1 id="foo">Foo</h1>
"
```
So, you are correct that the original was fine (actually, both are fine).... but marked is spitting out garbage.